requirements.txt

```diff
@@ -1,4 +1,4 @@
-astropy
 scipy
-numpy
+numpy>=1.16
```

Comment:
Is this one real or related to some other dep?

Reply:
astropy needs and, and for some reason Travis doesn't install it correctly in response on 3.7.  Might be something broken in the astropy reqs, not sure. If we can find a better way to fix this then we should.

Reply:
Try putting astropy above numpy in the file. Pip resolves versions as it goes. This should fix it.

Reply:
This didn't seem to help. For now I've reinstated the version requirement here.

Reply:
This might be some stupid travis thing, but I've put the requirements in place for now

Reply:
I'd like to leave this as-is for now, since it seems to work and shouldn't have any effect on real installations.